src/App.css

```
@@ -305,6 +305,105 @@
+
+.footer-content {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 100%;
+  padding: 0 2rem;
+  gap: 2rem;
+}
+
+.footer-copyright {
+  font-family: 'Montserrat', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Montserrat" (font-family-name-quotes)

(font-family-name-quotes)